clinica/pydra/machine_learning_spatial_svm/spatial_svm_cli.py

```diff
@@ -0,0 +1,98 @@
+from typing import Optional
+
+import click
+
+import clinica.pydra.engine_utils as pydra_utils
+import clinica.pydra.machine_learning_spatial_svm.pipeline as pydra_machine_learning_spatial_svm
+from clinica.pipelines import cli_param
+from clinica.pipelines.engine import clinica_pipeline
+
+pipeline_name = "pydra-machine-learning-prepare-spatial-svm"
+
+
+@clinica_pipeline
+@click.command(pipeline_name)
+@cli_param.argument.caps_directory
+@cli_param.argument.group_label
+@cli_param.argument.orig_input_data_ml
+@cli_param.option_group.custom_pipeline_options(
+    "Pipeline options if you use inputs from pet-volume pipeline"
+)
+@cli_param.option.acq_label
+@cli_param.option.suvr_reference_region
+@cli_param.option.use_pvc_data
+@cli_param.option_group.common_pipelines_options
+@cli_param.option.subjects_sessions_tsv
+@cli_param.option.working_directory
+@cli_param.option.n_procs
+@cli_param.option_group.advanced_pipeline_options
+@cli_param.option_group.option(
+    "-fwhm",
+    "--full_width_half_maximum",
+    default=4.0,
+    help=(
+        "Amount of regularization (in mm). In practice, we found the default value "
+        "(--full_width_half_maximum %(default)s) to be optimal. We therefore "
+        "do not recommend to change it unless you have a specific reason to do so."
+    ),
+)
+def cli(
+    caps_directory: str,
+    group_label: str,
+    orig_input_data_ml: str,
+    acq_label: Optional[str] = None,
+    suvr_reference_region: Optional[str] = None,
+    use_pvc_data: bool = False,
+    full_width_half_maximum: float = 4.0,
+    subjects_sessions_tsv: Optional[str] = None,
+    working_directory: Optional[str] = None,
+    n_procs: Optional[int] = None,
+) -> None:
+    """Prepare input data for SVM with spatial and anatomical regularization.
+
+       GROUP_LABEL is an user-defined identifier to target a specific group of subjects.
+
+       The third positional argument can be t1-volume to use tissue maps or pet-volume to use standardized uptake value ratio (SUVR) maps.
+
+    Prerequisites: You need to execute the t1-volume pipeline to run the pipeline on T1-weighted MRI data, and the t1-volume + pet-volume pipelines to apply it to PET data.
+
+    See https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/MachineLearning_PrepareSVM/"
+    """
+
+    from clinica.utils.exceptions import ClinicaException
```

Comment:
I just realized that we should adopt the same kind of logic as the one we used for `statistics_volume`. That is, move all the parameter checking logic in a `_check_pipeline_parameters` function 

https://github.com/aramis-lab/clinica/blob/a08d1c830c539ab9ec04a225a3907a8f5da1d044/clinica/pydra/statistics_volume/pipeline.py#L9-L21

which is then called at the beginning of `build_core_workflow`:

https://github.com/aramis-lab/clinica/blob/a08d1c830c539ab9ec04a225a3907a8f5da1d044/clinica/pydra/statistics_volume/pipeline.py#L102